hydra_base/util/__init__.py

```diff
@@ -26,6 +26,8 @@
 import json
 import six
 from .. import config
+import binascii
+import hashlib
 
 from collections import namedtuple
```

Comment:
All three imports unused - the two new ones are for future use with revising the hash type?